src/silx/gui/dialog/ColormapDialog.py

```diff
@@ -749,12 +755,26 @@ def _plotGammaMarkerConstraint(self, x, y):
             x = min(x, vmax)
         return x, y
 
+    def setDataInPlotMode(self, mode: Union[str, DataInPlotMode]):
```

Comment:
By adding a `from future import __annotations__` you can already spare the `from typing import Union` and write:
```suggestion
    def setDataInPlotMode(self, mode: str | DataInPlotMode):
```